.prettierrc.json

```diff
@@ -1,3 +1,9 @@
 {
-    "endOfLine": "auto"
-}
\ No newline at end of file
+  "semi": true,
+  "arrowParens": "always",
+  "printWidth": 120,
+  "bracketSpacing": true,
+  "trailingComma": "all",
+  "singleQuote": true,
```

Comment:
would actually prefer leaving that at double quotes to avoid too many unnecessary changes in the PR

Reply:
I'm also thinking the same to revert prettier to previous config. That will remove too many diffs